Sources/SwiftIfConfig/SyntaxLiteralUtils.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import SwiftSyntax
+
+extension BooleanLiteralExprSyntax {
+  var literalValue: Bool {
+    return booleanLiteral.tokenKind == .keyword(.true)
+  }
+}
+
+extension TupleExprSyntax {
+  /// Whether this tuple is a parenthesized expression, e.g., (x).
+  var isParentheses: Bool {
+    guard elements.count == 1, let element = elements.first else { return false }
+    return element.label == nil
+  }
+}
+
+extension TupleExprElementListSyntax {
+  /// If this list is a single, unlabeled expression, return it.
+  var singleUnlabeledExpression: ExprSyntax? {
+    guard count == 1, let element = first else { return nil }
+    return element.expression
+  }
+}
+
+extension ExprSyntax {
+  /// Whether this is a simple identifier expression and, if so, what the identifier string is.
+  var simpleIdentifierExpr: String? {
```

Comment:
I find it easier to work with the string, because that's what clients want. This is in the same vein as the other comment about wanting to have a central "get the identifier for this token" API to use in various places.